ramalama/engine.py

```diff
@@ -0,0 +1,272 @@
+import glob
+import json
+import os
+import subprocess
+import sys
+
+from ramalama.common import (
+    check_nvidia,
+    exec_cmd,
+    get_accel_env_vars,
+    perror,
+    podman_machine_accel,
+    run_cmd,
+)
+from ramalama.console import EMOJI
+
+
+class Engine:
+
+    def __init__(self, args):
+        self.exec_args = [
+            args.engine,
+            "run",
+            "--rm",
+        ]
+        self.use_docker = os.path.basename(args.engine) == "docker"
```

Comment:
Super minor, but could call os.path.basename once. Micro-optimization

Reply:
Let's merge and do that as a cleanup.